File: Business/TravelRepublic.Business.Common/Responses/HotelSearchFilterResponse.cs
using Eml.Mediator.Contracts;
using System.Collections.Generic;
using TravelRepublic.Business.Common.Dto;

namespace TravelRepublic.Business.Common.Responses
{
    public class HotelSearchFilterResponse : IResponse
    {
        public IEnumerable<StarFilter> StarFilters { get; }

        public double RatingMin { get; }

        public double RatingMax { get; }

        public double CostMin { get; }

        public double CostMax { get; }

        public HotelSearchFilterResponse(IEnumerable<StarFilter> starFilters, double ratingMin, double ratingMax, double costMin, double costMax)
        {
            StarFilters = starFilters;
            RatingMin = ratingMin;
            RatingMax = ratingMax;
            CostMin = costMin;
            CostMax = costMax;
        }
    }
}

File: Hosts/TravelRepublic.Spa/src/app/modules/hotels/responses/hotel-search-filter-response.ts
import { StarFilter } from '../dto/star-filter';

export class HotelSearchFilterResponse {
    constructor(public starFilters: StarFilter[] = [],
                public ratingMin = 0,
                public ratingMax = 100,
                public costMin = 0,
                public costMax = 0) { }
}
